en/debug/eclipse_jlink.md

```diff
@@ -78,6 +78,16 @@ For more information, see: [https://gnu-mcu-eclipse.github.io/debug/jlink/instal
    ![Eclipse: GDB Segger Debug config: debugger tab](../../assets/debug/eclipse_settings_debug_config_gdb_segger_build_config_debugger_tab.png)
    ![Eclipse: GDB Segger Debug config: startup tab](../../assets/debug/eclipse_settings_debug_config_gdb_segger_build_config_startup_tab.png)
 
+## SEGGER Task-aware debugging
```

Comment:
Is this something that should also be added to the VS Studio docs in some way?

Reply:
VS Code can do this as well but could also be automated see
https://github.com/PX4/PX4-Autopilot/pull/20673#issuecomment-1329209245
However I don't use VS Code that often myself so someone more experienced could take a look at that.

Reply:
I'll be going through it shortly.